docs/data/material/varnish/varnish-with-react-router/SimpleNav.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { Header, VarnishApp } from '@allenai/varnish';
+
+import { BrowserRouter, NavLink} from "react-router-dom";
+import { StaticRouter } from 'react-router-dom/server';
+
+// This is something we have to do to play nicely with Material UI's docs pages. 
+// This should not be needed when utilizing React Router on other web pages.
+function Router(props: { children?: React.ReactNode }) {
+  const { children } = props;
+  if (typeof window === 'undefined') {
+    return <StaticRouter location="/">{children}</StaticRouter>;
+  }
+
+  return <BrowserRouter>{children}</BrowserRouter>;
+}
+
+export default function Demo() {
+  return (
+    <VarnishApp>
+      <div style={{ width: '100%' }}>
```

Comment:
We should start to use Varnish2's 'Content' here instead of div full width
let me know if it does nor does not work